app/controllers/users_controller.rb

```diff
@@ -1,5 +1,17 @@
 class UsersController < ApplicationController
-  def show; end
+  def show
+    conn = Faraday.new(url: 'https://api.github.com') do |faraday|
+      faraday.headers['Authorization'] = "token #{current_user.token}"
+    end
+
+    response = conn.get('user/repos')
+    json = JSON.parse(response.body, symbolize_names: true)
+
+    @repos = []
```

Comment:
I like that we got this working and will get it refactored after tomorrow's lesson